lib/util/index.js

```diff
@@ -535,32 +535,21 @@ function valueToJSON(value) {
  * @returns {object}
  */
 function createBandwidthProfilePayload(bandwidthProfile) {
-  const payload = {};
-  if (typeof bandwidthProfile.video !== 'object') {
+  return typeof bandwidthProfile.video === 'object' ? [
+    ['dominantSpeakerPriority', 'string', 'active_speaker_priority'],
+    ['maxSubscriptionBitrate', 'number', 'max_subscription_bandwidth', x => Math.round(x / 1024)],
+    ['maxTracks', 'number', 'max_tracks'],
+    ['mode', 'string'],
+    ['renderDimensions', 'object', 'render_dimensions', createRenderDimensionsPayload]
+  ].reduce((payload, [prop, dataType, payloadProp = prop, transform = x => x]) => {
+    const value = bandwidthProfile.video[prop];
+    if (typeof value === dataType) {
+      payload.video = Object.assign({
```

Comment:
```suggestion
      payload.video[payloadProp] = transform(value)
```

Reply:
@makarandp0 Then we would have to add `payload.video = payload.video || {}` everywhere.